pkg/k8s/apis/cilium.io/v2/register.go

```diff
@@ -34,7 +34,7 @@ const (
 	//
 	// Maintainers: Run ./Documentation/check-crd-compat-table.sh for each release
 	// Developers: Bump patch for each change in the CRD schema.
-	CustomResourceDefinitionSchemaVersion = "1.22.4"
+	CustomResourceDefinitionSchemaVersion = "1.22.5"
```

Comment:
We had something that required a bump?

Reply:
Evidently yes, the script picked it up. Commit f04540885c84 ( Backport #16035, PR https://github.com/cilium/cilium/pull/15911).

Reply:
Just a description comment so nothing notable but the script doesn't care what the change is, if there's any change, it needs a version bump:

```
diff --git a/examples/crds/ciliumnodes.yaml b/examples/crds/ciliumnodes.yaml
index e115f68cb7c0..b2413484e744 100644
--- a/examples/crds/ciliumnodes.yaml
+++ b/examples/crds/ciliumnodes.yaml
@@ -305,8 +305,8 @@ spec:
                         \n More details: https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-eni.html"
                       properties:
                         addresses:
-                          description: Addresses is the list of all IPs associated
-                            with the ENI, including all secondary addresses
+                          description: Addresses is the list of all secondary IPs
+                            associated with the ENI
                           items:
                             type: string
                           type: array
```

Reply:
nice to see the script finally working :tada: